api/edge/v1alpha1/types.go

```diff
@@ -171,6 +173,133 @@ type PlacementList struct {
 	Items           []Placement `json:"items"`
 }
 
+/******************** End Placement Type *********************/
+
+/******************** Start PlacementDecision Type *********************/
+
+// PlacementDecision is mapped 1:1 to a single Placement resource.
+// The decision resource reflects the resolution of the Placement's selectors,
+// and explicitly reflects which resources should go to what destinations.
+//
+// +genclient
+// +genclient:nonNamespaced
+// +kubebuilder:object:root=true
+// +kubebuilder:resource:scope=Cluster,shortName={pd}
+type PlacementDecision struct {
+	metav1.TypeMeta `json:",inline"`
+	// Standard object metadata.
+	// More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+	// +optional
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	// `spec` explicitly describes a desired binding between workloads and Locations.
+	// It reflects the resolution of a Placement's selectors.
+	Spec PlacementDecisionSpec `json:"spec,omitempty"`
+
+	// `status` describes the status of the process of binding workloads to Locations.
+	// +optional
+	Status PlacementDecisionStatus `json:"status,omitempty"`
+}
+
+// PlacementDecisionSpec holds a list of resources and a list of destinations which are the resolution
+// of a Placement's `what` and `where`: what resources to propagate and to where.
+// All resources present in this spec are propagated to all destinations present.
+type PlacementDecisionSpec struct {
+	// `Workload` is a collection of namespaced-scoped resources and a collection of cluster-scoped resources to be propagated to destination clusters.
+	Workload PlacementDecisionSpecResources `json:"workload,omitempty"`
+
+	// `destinations` is a list of cluster-identifiers that the resources should be propagated to.
+	Destinations []Destination `json:"destinations,omitempty"`
+}
+
+// PlacementDecisionSpecResources explicitly defines the resources to be down-synced.
+// The ClusterScope list defines the cluster-scope resources, while NamespacedObjects packs individual resources
+// identifiable by namespace & name.
+type PlacementDecisionSpecResources struct {
```

Comment:
I think we could use a shorter name, describing what one of these is rather than where it gets used. And that is a clearer way to name something anyway, right? How about `DownsyncObjectReferences` ? Or `DownsyncWorkloadParts` ?

Reply:
done. changed to `DownsyncObjectsReferences`